components/brave_wallet/browser/solana_transaction_unittest.cc

```diff
@@ -209,6 +209,18 @@ TEST_F(SolanaTransactionUnitTest, GetSignedTransaction) {
   expected_tx = base::Base64Encode(expected_bytes);
   EXPECT_EQ(transaction2.GetSignedTransaction(keyring_service()), expected_tx);
 
+  // Test when num of signatures available is less than signers.size in message,
+  // should use the actual number of signatures available.
+  sign_tx_param->signatures.pop_back();
+  transaction2.set_sign_tx_param(sign_tx_param.Clone());
+  expected_bytes = std::vector<uint8_t>({1});  // # of signature
+  expected_bytes.insert(expected_bytes.end(), signature.begin(),
+                        signature.end());
+  expected_bytes.insert(expected_bytes.end(), message_bytes.begin(),
+                        message_bytes.end());
+  EXPECT_EQ(transaction2.GetSignedTransaction(keyring_service()),
+            base::Base64Encode(expected_bytes));
+
```

Comment:
can we also add a test that is num of signatures larger than signers.size? And final signature size should be signers.size.

Reply:
https://github.com/brave/brave-core/compare/4ee63086cdc0d66ea8b7845fceb6eecead0567ce..42a28ae80d207a4b3b9cb1e82133028e8f30559d